app/views/events/index.html.slim

```diff
@@ -1,9 +1,13 @@
 - @hash.each do |month,events|
   .row.month-events
-    div.month #{month}
+    div
+    - printed = false
```

Comment:
I've noticed that when there's no confirmed events name of the month will be displayed anyway. So no-admin users can see empty month. That bool helped me to prevent it.